shoop/themes/classic_gray/__init__.py

```diff
@@ -4,8 +4,9 @@
 #
 # This source code is licensed under the AGPLv3 license found in the
 # LICENSE file in the root directory of this source tree.
-
 from shoop.apps import AppConfig
+from shoop.themes.classic_gray.views.product_preview import product_quick_view
```

Comment:
You can't safely import views (that eventually do any model imports) here.

Move these imports to `get_view`?

You can even do

```python
def get_view(self, view_name):
    import shoop.themes.classic_gray.views as views
    return getattr(views, view_name, None)
```

and in `views/__init__.py` import the views with the xtheme extra view names:

```python
from .product_preview import product_quick_view as preview
from .basket import basket_partial
```

:sunglasses: